external/apb/data-sync.html

```diff
@@ -1 +1 @@
-<meta http-equiv="refresh" content="0; URL=''" />
\ No newline at end of file
+<meta http-equiv="refresh" content="0; URL='./../../aerogear/latest/data_sync.html'" />
```

Comment:
hi @jhellar the redirect works fine but based on other files in the latest directory i think it should be `data-sync.html` rather than `data_sync.html`. Very small issue and it would mean renaming the file [here](https://github.com/aerogear/mobile-docs/blob/sync/modules/ROOT/pages/data_sync.adoc) @finp is it important to have a syntax for these files?

Reply:
I don't think this is critical, but we should try to be consistent @jhellar